cmd/manager/main.go

```diff
@@ -171,3 +185,42 @@ func main() {
 		klog.Fatalf("Error starting manager: %v", err)
 	}
 }
+
+// newConfigManagedClient returns a controller-runtime client that can be used to access the openshift-config-managed
+// namespace.
+func newConfigManagedClient(config *rest.Config) (runtimeclient.Client, manager.Runnable, error) {
+	scheme := runtime.NewScheme()
+	if err := corev1.AddToScheme(scheme); err != nil {
+		return nil, nil, err
+	}
+
+	mapper, err := apiutil.NewDynamicRESTMapper(config)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	cacheOpts := cache.Options{
+		Scheme:    scheme,
+		Mapper:    mapper,
+		Namespace: awsclient.KubeCloudConfigNamespace,
+	}
+	cache, err := cache.New(config, cacheOpts)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	startCache := manager.RunnableFunc(func(ctx context.Context) error {
+		return cache.Start(ctx)
+	})
```

Comment:
The `cache` already implements the runnable here, any reason not to just return the cache directly under the Runnable interface?

Reply:
Makes sense. I was initially doing more work in this runnable to wait for the cache to be synced, but it didn't make sense to that in there. I will remove this and return `cache` directly.